### PR TITLE
 Grid 

### PR DESCRIPTION
**Pull request by khushijohri2001**
Please review my code for my Component Library's Grid Component.

The page includes:

Header with Girlifycoder Logo, Heading, and Github link
The navigation bar on the left
Sidebar on the right
Details about the Component and snippet

https://user-images.githubusercontent.com/65448358/160250927-26770040-4c61-40a8-b87a-555390bb9ce7.mp4

### CODE REVIEW

File: grids/styles2-grids.css [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+}
+
+/* grid-2 */
+.deliver-container {
+  display: grid;
+  grid-auto-flow: column;
+  border-radius: 0.5rem;
+  padding: 1rem;
+  background-color: #029aa8;
+  color: white;
```
Comment (pnchinmay) [inline grids/styles2-grids.css]: Use hex code

File: grids/styles2-grids.css [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+  margin: 1rem 0rem;
+}
+
+/* grid-2 */
+.deliver-container {
+  display: grid;
+  grid-auto-flow: column;
+  border-radius: 0.5rem;
+  padding: 1rem;
+  background-color: #029aa8;
```
Comment (pnchinmay) [inline grids/styles2-grids.css:36]: Color variables are recommended

**Review by pnchinmay — APPROVED** [review]

File: grids/styles2-grids.css [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+.grids-container {
+  margin: 3rem;
+}
+
+.grids-main-heading {
+  color: yellow;
+}
+.grids-snippet iframe {
+  margin: 1rem 0rem;
+  width: 564px;
```
Comment (i-am-a-codegirl) [inline grids/styles2-grids.css:11]: use rems instead of px

**Review by i-am-a-codegirl — APPROVED** [review]

File: grids/styles2-grids.css (hunk 3 above)
Comment (khushijohri2001) [reply grids/styles2-grids.css:11]: okay

File: grids/styles2-grids.css (hunk 2 above)
Comment (khushijohri2001) [reply grids/styles2-grids.css:36]: done

**Pull request merged**